buildbot/master/init.sls

```diff
@@ -90,7 +90,7 @@ buildbot-github-listener:
 
 remove-old-build-logs:
   cron.present:
-    - name: 'find {{ common.servo_home }}/buildbot/master/*/*.bz2 -mtime +5 -delete'
+    - name: 'find {{ common.servo_home }}/buildbot/master/*/* -mtime +5 -delete'
```

Comment:
Getting rid of the second `/*` at the end should probably do the trick. For the bz2 cron, we can add a `-name '*.bz2'` as well, and make a separate one for all logs.